src/bin/cql-stress-cassandra-stress/settings/mod.rs

```diff
@@ -22,7 +23,7 @@ use self::option::SchemaOption;
 
 pub struct CassandraStressSettings {
     pub command: Command,
-    pub params: CommandParams,
+    pub command_params: CommandParams,
```

Comment:
This rename should probably be a separate commit. It looks like you really are mixing multiple things in the first commit, but you shouldn't.